sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_call_connection.py

```diff
@@ -74,9 +74,8 @@ def cancel_all_media_operations(
         **kwargs  # type: Any
     ):  # type: (...) -> CancelAllMediaOperationsResult
 
-        if operation_context is not None:
-            kwargs['operation_context'] = operation_context
-        request = CancelAllMediaOperationsRequest(**kwargs)
+        request = CancelAllMediaOperationsRequest(operation_context=operation_context,
```

Comment:
The is not needed, because the the code was for python2 support, please check out this model class https://github.com/Azure/azure-sdk-for-python/blob/55839d9ec86f434b7f91d0027f5ac3094de68bc2/sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_generated/models/_models.py#L230-L247

Reply:
But I'll add a wrap static method to ensure it's consistent with other apis which need to build a **Request** object as parameters in the http request.